---
title: Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for UGT1A1 and atazanavir prescribing.
id: 11000
---

# General

## Guideline Title

Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for _UGT1A1_ and atazanavir prescribing.

## Bibliographic Source(s)

- Gammal RS, Court MH, Haidar CE, Iwuchukwu OF, Gaur AH, Alvarellos M, Guillemette C, Lennox JL, Whirl-Carrillo M, Brummel SS, Ratain MJ, Klein TE, Schackman BR, Caudle KE, Haas DW. Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for UGT1A1 and atazanavir prescribing. Clin Pharmacol Ther. 2016 Apr;99(4):363-9. [40 references] [ PubMed ](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=26417955)

## Guideline Status



This is the current release of the guideline.

This guideline meets NGC's 2013 (revised) inclusion criteria.

# Recommendations

## Major Recommendations



The strength of therapeutic recommendations ( **Strong, Moderate, Optional** ) is defined at the end of the "Major Recommendations" field. 


###  Genetic Test Interpretation 


Clinical laboratories generally report uridine diphosphate glucuronosyltransferase ( _UGT_ ) _1A1_ genotype assay results for the more frequent alleles, using either the star (*) allele nomenclature and/or the number of thymine-adenine (TA) repeats in the _UGT1A1_ gene promoter region. Each named * allele is defined by one or more specific polymorphisms (see Supplemental Table S1 [see the "Availability of the Companion documents" field]). The level of UGT1A1 activity associated with the most frequent allelic variants is summarized in Supplemental Table S2. Genotyping rs8175347 for the number of TA repeats allows assignment to _UGT1A1*28_ (TA  7  ), _UGT1A1*36_ (TA  5  ), _UGT1A1*37_ (TA  8  ), or _UGT1A1*1_ (TA  6  , reference genotype). Because rs887829 is in almost complete linkage with rs8175347 (r  2  â‰… 0.99), metabolizer status may also be inferred based on rs887829. The table below summarizes the assignment of the likely UGT1A1 phenotype based on * allele and number of TA repeats. 


Alleles of _UGT1A1_ have been characterized in various geographically, racially, and ethnically diverse populations (see Supplemental Table S3). The _UGT1A1*6_ allele (rs4148323, 211G >A) is associated with reduced UGT1A1 enzyme function and is found almost exclusively among individuals of Asian descent. In general, genotyping tests do not identify very rare or _de novo_ variants. 


####  Table: Assignment of Likely UGT1A1 Phenotypes Based on Genotypes 
<table border="1" cellpadding="3" cellspacing="3" summary="Table: Assignment of Likely UGT1A1 Phenotypes Based on Genotypes">
 <thead>
  <tr>
   <th style="font-weight: bold; text-align: center;" valign="top">
    Likely Phenotype
   </th>
   <th style="font-weight: bold; text-align: center;" valign="top">
    Genotypes
   </th>
   <th style="font-weight: bold; text-align: center;" valign="top">
    Examples of Diplotypes
   </th>
  </tr>
 </thead>
 <tbody>
  <tr>
   <td valign="top">
    Extensive metabolizer
   </td>
   <td valign="top">
    An individual carrying two reference
    <sup>
     b
    </sup>
    function (*
    <em>
     1
    </em>
    )
    <sup>
     c
    </sup>
    and/or increased function alleles (*
    <em>
     36
    </em>
    ). Alternatively identified by homozygosity for rs887829 C/C.
   </td>
   <td valign="top">
    *
    <em>
     1
    </em>
    /*
    <em>
     1
    </em>
    ; *
    <em>
     1
    </em>
    /*
    <em>
     36
    </em>
    ; *
    <em>
     36
    </em>
    /*
    <em>
     36
    </em>
    ; rs887829 C/C
   </td>
  </tr>
  <tr>
   <td valign="top">
    Intermediate metabolizer
   </td>
   <td valign="top">
    An individual carrying one reference
    <sup>
     b
    </sup>
    function (*
    <em>
     1
    </em>
    )
    <sup>
     c
    </sup>
    or increased function allele (*
    <em>
     36
    </em>
    ) plus one decreased function allele (*
    <em>
     6
    </em>
    , *
    <em>
     28
    </em>
    , *
    <em>
     37
    </em>
    ). Alternatively identified by heterozygosity for rs887829 C/T.
   </td>
   <td valign="top">
    *
    <em>
     1
    </em>
    /*
    <em>
     28
    </em>
    ; *
    <em>
     1
    </em>
    /*
    <em>
     37
    </em>
    ; *
    <em>
     36
    </em>
    /*
    <em>
     28
    </em>
    ; *
    <em>
     36
    </em>
    /*
    <em>
     37
    </em>
    ; rs887829 C/T; *
    <em>
     1
    </em>
    /*
    <em>
     6
    </em>
   </td>
  </tr>
  <tr>
   <td valign="top">
    Poor metabolizer
   </td>
   <td>
    An individual carrying two decreased function alleles (*
    <em>
     6
    </em>
    , *
    <em>
     28
    </em>
    , *
    <em>
     37
    </em>
    ). Alternatively identified by homozygosity for rs887829 T/T (*
    <em>
     80
    </em>
    /*
    <em>
     80
    </em>
    ).
   </td>
   <td valign="top">
    *
    <em>
     28
    </em>
    /*
    <em>
     28
    </em>
    ; *
    <em>
     28
    </em>
    /*
    <em>
     37
    </em>
    ; *
    <em>
     37
    </em>
    /*
    <em>
     37
    </em>
    ; rs887829 T/T (*
    <em>
     80
    </em>
    /*
    <em>
     80
    </em>
    ); *
    <em>
     6
    </em>
    /*
    <em>
     6
    </em>
    <sup>
     a
    </sup>
   </td>
  </tr>
 </tbody>
</table>


UGT, uridine diphosphate glucuronosyltransferase 


#####  aHomozygosity for UGT1A1*6, which occurs almost exclusively in individuals of Asian descent, is associated with Gilbert's syndrome. However, at this time it is unclear if patients with this diplotype are at increased risk of severe atazanavir-associated hyperbilirubinemia. 


#####  b"Reference" function refers to the UGT1A1 alleles to which other alleles are compared. 


c  The reference function *1 allele is fully functional and refers to the rs8175347 TA  6  allele. 


###  Therapeutic Recommendations 


#####  Atazanavir-associated indirect hyperbilirubinemia does not indicate hepatic injury, but some patients are not prescribed atazanavir to avoid the possibility of jaundice. Implications of UGT1A1 genotype data for prescribing of atazanavir, boosted with either ritonavir or cobicistat, may be influenced by several factors. These include consequences of jaundice for the particular patient (e.g., workers who frequently interact with the public), access to alternative protease inhibitor antiretrovirals (e.g., darunavir), and whether the provider finds it useful to monitor atazanavir-induced changes in bilirubin concentrations to assess adherence. Recommendations are provided in the table below. 


####  Table: Recommended Use of Atazanavir (Boosted with either Ritonavir or Cobicistat*) by UGT1A1 Phenotype 
<table border="1" cellpadding="3" cellspacing="3" summary="Table: Recommended Use of Atazanavir (Boosted with either Ritonavir or Cobicistat*) by &lt;em&gt;UGT1A1&lt;/em&gt; Phenotype">
 <thead>
  <tr>
   <th style="font-weight: bold; text-align: center;" valign="top">
    Phenotype
   </th>
   <th style="font-weight: bold; text-align: center;" valign="top">
    Implications for Phenotypic Measures
   </th>
   <th style="font-weight: bold; text-align: center;" valign="top">
    Dosing Recommendations
   </th>
   <th style="font-weight: bold; text-align: center;" valign="top">
    Classification of Recommendations
   </th>
  </tr>
 </thead>
 <tbody>
  <tr>
   <td valign="top">
    Extensive metabolizer
   </td>
   <td valign="top">
    Reference
    <sup>
     a
    </sup>
    UGT1A1 activity; very low likelihood of bilirubin-related discontinuation of atazanavir.
   </td>
   <td valign="top">
    There is no need to avoid prescribing of atazanavir based on
    <em>
     UGT1A1
    </em>
    genetic test result. Inform the patient that some patients stop atazanavir because of jaundice (yellow eyes and skin), but that this patient's genotype makes this unlikely (less than about a 1 in 20 chance of stopping atazanavir because of jaundice).
   </td>
   <td valign="top">
    Strong
   </td>
  </tr>
  <tr>
   <td valign="top">
    Intermediate metabolizer
   </td>
   <td valign="top">
    Somewhat decreased UGT1A1 activity; low likelihood of bilirubin-related discontinuation of atazanavir.
   </td>
   <td>
    There is no need to avoid prescribing of atazanavir based on UGT1A1 genetic test result. Inform the patient that some patients stop atazanavir because of jaundice (yellow eyes and skin), but that this patient's genotype makes this unlikely (less than about a 1 in 20 chance of stopping atazanavir because of jaundice).
   </td>
   <td valign="top">
    Strong
   </td>
  </tr>
  <tr>
   <td valign="top">
    Poor metabolizer
   </td>
   <td valign="top">
    Markedly decreased UGT1A1 activity; high likelihood of bilirubin-related discontinuation of atazanavir.
   </td>
   <td valign="top">
    Consider an alternative agent particularly where jaundice would be of concern to the patient.
    <br/>
    <br/>
    If atazanavir is to be prescribed, there is a high likelihood of developing jaundice that will result in atazanavir discontinuation (at least 20% and as high as 60%).
   </td>
   <td valign="top">
    Strong
   </td>
  </tr>
 </tbody>
</table>


UGT, uridine diphosphate glucuronosyltransferase 


*All studies correlating _UGT1A1_ genotypes with atazanavir adverse events have involved ritonavir boosting. However, concentration-time profiles are equivalent when boosted with either cobicistat or ritonavir, and bilirubin-related adverse events including discontinuation of atazanavir occur in a similar percentage of patients prescribed atazanavir with cobicistat or ritonavir. Associations between _UGT1A1_ genotype, bilirubin elevations, and atazanavir/r discontinuation therefore almost certainly translate to atazanavir/cobicistat. 


#####  a"Reference" function refers to the UGT1A1 allele to which other alleles are compared. 


A _UGT1A1_ genotype is most helpful if available before atazanavir is prescribed. If noticeable jaundice does not develop while taking atazanavir chronically (or develops but is not bothersome), then the risk for bilirubin-related atazanavir discontinuation is probably low regardless of _UGT1A1_ genotype. 


For individuals carrying two _UGT1A1_ decreased function alleles (i.e., _UGT1A1*28/*28_ , _UGT1A1*28/*37_ , _UGT1A1*37/*37_ , or rs887829 T/T), the likelihood of bilirubin-related atazanavir discontinuation is substantial. Before such individuals are prescribed atazanavir (boosted with either ritonavir or cobicistat), all such patients should be advised about the substantial likelihood of developing jaundice. Prescribing atazanavir to such individuals should generally be avoided unless the patient does not consider jaundice to be a concern, or there are other compelling reasons to prescribe atazanavir. 


For individuals carrying fewer than two _UGT1A1_ decreased function alleles (i.e., _*1/*28, *1/*37, *36/*28, *36/*37,_ rs887829 C/C or rs887829 C/T), the likelihood of bilirubin-related atazanavir discontinuation is low. This risk is extremely low for individuals carrying no _UGT1A1_ decreased function alleles (i.e., _UGT1A1*1/*1, UGT1A1*1/*36, UGT1A1*36/*36_ , or rs887829 C/C). Among patients with extensive metabolizer UGT1A1 phenotypes it may not be necessary to discuss the possibility of jaundice with atazanavir. This decision about whether to discuss possible jaundice should be based on the clinical situation and provider judgment. If advice is offered, such discussion may note that the likelihood of developing jaundice that would require discontinuation of atazanavir is very low. 


####  Recommendations for Pediatrics 


At the time of the guideline's writing there are no pediatric data regarding associations between _UGT1A1_ genotypes and likelihood of bilirubin-related discontinuation of atazanavir. However, _UGT1A1_ genotypes are expected to affect atazanavir-related hyperbilirubinemia similarly in adults and children Therefore, recommendations for adults may be directly adapted to pediatric patients. 


####  Recommendations for Incidental Findings 


Individuals who are homozygous for _UGT1A1*28_ or _UGT1A1*6_ are very likely to have Gilbert syndrome. Knowing an individual's _UGT1A1_ genotype prior to prescribing may have implications for selection and dosing for drugs known to be UGT1A1 substrates or inhibitors, such as irinotecan and nilotinib. 


###  Definitions 


####  Strength of Therapeutic Recommendations 


**Strong** : The evidence is high quality and the desirable effects clearly outweigh the undesirable effects. 


**Moderate** : There is a close or uncertain balance as to whether the evidence is high quality and the desirable clearly outweigh the undesirable effects. 


**Optional** : The desirable effects are closely balanced with undesirable effects and there is room for differences of opinion as to the need for the recommended course of action. 
## Clinical Algorithm(s)



The following algorithms are provided in the Supplemental Material (see the "Availability of Companion Documents" field):

  * _UGT1A1_ Pharmacogenetic Test Result: Clinical Implementation Workflow for EHR 
  * _UGT1A1_ Genotype and Atazanavir: Point of Care Clinical Decision Support 



# Scope

## Disease/Condition(s)



  * Human immunodeficiency virus (HIV) infection 
  * Hyperbilirubinemia 
  * Jaundice 



## Guideline Category

- Evaluation
- Risk Assessment

## Clinical Specialty

- Allergy and Immunology
- Infectious Diseases
- Medical Genetics
- Pharmacology

## Intended Users

- Advanced Practice Nurses
- Pharmacists
- Physician Assistants
- Physicians

## Guideline Objective(s)



To provide information to allow the interpretation of clinical _UGT1A1_ genotype tests so that the results can be used to inform the prescribing of atazanavir

## Target Population



Individuals with human immunodeficiency virus (HIV) infection considering therapy with atazanavir

## Interventions and Practices Considered



Use of _UGT1A1_ genotyping to guide therapeutic decision-making regarding prescribing of atazanavir

## Major Outcomes Considered



Effect of _UGT1A1_ genotype on atazanavir (risk for atazanavir associated hyperbilirubinemia and related atazanavir discontinuation) or _in vivo_ clinical outcome for atazanavir/ritonavir (r) and _in vivo_ pharmacokinetics and pharmacodynamics (PK/PD) for atazanavir/r

# Methodology

## Methods Used to Collect/Select the Evidence

- Searches of Electronic Databases

## Description of Methods Used to Collect/Select the Evidence



 **Literature Review**

The authors searched the PubMed database (1966 to December 2015) for keywords (UGT1A1 or UGT1A* or Gilbert or hyperbilirubinemia or jaundice or UGT1A1 and discontinuation) AND (atazanavir). Using these search terms, 212 publications were identified. In addition, studies annotated in PharmGKB ([http://www.pharmgkb.org](https://www.pharmgkb.org/ "PharmGKB Web site")) were identified. Study inclusion criteria included publications that included analyses of the effect of _UGT1A1_ on clinical outcomes of atazanavir use (hyperbilirubinemia, jaundice, or drug discontinuation). Non-English manuscripts were excluded.

## Number of Source Documents



Following application of these inclusion criteria, 24 publications were reviewed and included in the evidence table (Supplemental Table S4 [see the "Availability of Companion Documents" field]).

## Methods Used to Assess the Quality and Strength of the Evidence

- Weighting According to a Rating Scheme (Scheme Given)

## Rating Scheme for the Strength of the Evidence

<div class="content_para"><p><strong><span style="text-decoration: underline;">Levels of Evidence</span></strong></p>
<p><strong>High</strong>: Evidence includes consistent results from well-designed, well-conducted studies.</p>
<p><strong>Moderate</strong>: Evidence is sufficient to determine effects, but the strength of the evidence is limited by the number, quality, or consistency of the individual studies; generalizability to routine practice; or indirect nature of the evidence.</p>
<p><strong>Weak</strong>: Evidence is insufficient to assess the effects on health outcomes because of limited number or power of studies, important flaws in their design or conduct, gaps in the chain of evidence, or lack of information.</p></div>

## Methods Used to Analyze the Evidence

- Systematic Review with Evidence Tables

## Description of the Methods Used to Analyze the Evidence



The Clinical Pharmacogenetics Implementation Consortium's (CPIC's) dosing recommendations are based on weighting the evidence from a combination of preclinical, functional, and clinical data, as well as on some existing disease-specific consensus guidelines. Some of the factors that have been taken into account for this guideline include _in vivo_ clinical outcome for atazanavir/ritonavir (r) and _in vivo_ pharmacokinetics and pharmacodynamics (PK/PD) for atazanavir/r.

The evidence summarized in Supplemental Table S4 (see the "Availability of Companion Documents" field) is graded on a scale of high, moderate, and weak, based upon the level of evidence (see the "Rating Scheme for the Strength of the Evidence" field).

## Methods Used to Formulate the Recommendations

- Expert Consensus

## Description of Methods Used to Formulate the Recommendations



Overall, the dosing recommendations are simplified to allow rapid interpretation by clinicians. The authors chose to use a slight modification of a transparent and simple system for just three categories of recommendations adopted from the rating scale for evidence-based recommendations on the use of antiretroviral agents (see the "Rating Scheme for the Strength of the Recommendations").

## Rating Scheme for the Strength of the Recommendations



 **Strength of Therapeutic Recommendations**

**Strong** : The evidence is high quality and the desirable effects clearly outweigh the undesirable effects.

**Moderate** : There is a close or uncertain balance as to whether the evidence is high quality and the desirable clearly outweigh the undesirable effects.

**Optional** : The desirable effects are closely balanced with undesirable effects and there is room for differences of opinion as to the need for the recommended course of action.

## Cost Analysis



Analyses of cost-effectiveness are beyond the scope of this guideline.

## Method of Guideline Validation

- Peer Review

## Description of Method of Guideline Validation



Not stated

# Evidence Supporting the Recommendations

## Type of Evidence Supporting the Recommendations



The type of supporting evidence is identified and graded for each recommendation (see the "Major Recommendations" field).

# Benefits/Harms of Implementing the Guideline Recommendations

## Potential Benefits



The benefit of _UGT1A1_ genotype would be to determine the individual's likelihood of bilirubin-related discontinuation of atazanavir prior to beginning therapy. This may allow atazanavir to be prescribed to patients at low risk for bilirubin-related discontinuation and avoided in patients at high risk.

## Potential Harms



  * In April 2015, the US Department of Health and Human Services Panel on Antiretroviral Guidelines for Adults and Adolescents downgraded regimens containing atazanavir/ritonavir (r) from recommended to alternative status based on a large comparative trial showing the rate of toxicity-related discontinuation was greater with the atazanavir/r than with either darunavir/r or raltegravir, each given with tenofovir/emtricitabine. In 2015, the FDA approved a coformulated tablet comprising atazanavir with cobicistat, which was likewise relegated to alternative status. With once-daily atazanavir 300 mg plus ritonavir 100 mg, the frequency of grade 3 or higher bilirubin elevations (at least 2.5 times the upper limit of normal) is ~40%, and of grade 4 (at least 5 times the upper limit of normal) bilirubin elevation is ~4-8%. Among children and adolescents in a trial that involved atazanavir, 9% had a bilirubin value â‰¥5.1 times the upper limit of normal and 1.4% experienced jaundice. 
  * There is little apparent risk of _UGT1A1_ genotyping that results in a recommendation to avoid atazanavir, as alternative protease inhibitor-containing regimens are comparable in terms of efficacy and pill burden, although costs may vary depending on the payer. 
  * Other possible limitations of _UGT1A1_ genotyping include laboratory error and an incomplete _UGT1A1_ genetic profile, as many tests only report results for _UGT1A1*28_. As individuals' genotypes do not change over time, genotyping errors could remain in the medical record for the lifetime of the patient. 



# Qualifying Statements

## Qualifying Statements



**Disclaimer**

Clinical Pharmacogenetics Implementation Consortium (CPIC) guidelines reflect expert consensus based on clinical evidence and peer-reviewed literature available at the time they are written, and are intended only to assist clinicians in decision-making, as well as to identify questions for further research. New evidence may have emerged since the time a guideline was submitted for publication. Guidelines are limited in scope and are not applicable to interventions or diseases not specifically identified. Guidelines do not account for all individual variations among patients and cannot be considered inclusive of all proper methods of care or exclusive of other treatments. It remains the responsibility of the health care provider to determine the best course of treatment for the patient. Adherence to any guideline is voluntary, with the ultimate determination regarding its application to be solely made by the clinician and the patient. CPIC assumes no responsibility for any injury to persons or damage to property related to any use of CPIC's guidelines, or for any errors or omissions.

**Caveats: Appropriate Use and/or Potential Misuse of Genetic Tests**

A systematic review of the literature concluded that homozygosity of _UGT1A1*28_ is a risk factor for occurrence of severe atazanavir-associated unconjugated hyperbilirubinemia, with a pooled positive predictive value of 40.3% and a pooled negative predictive value of 88.1%. Bilirubin-related discontinuation of atazanavir through 96 weeks is strongly associated with rs887829 T/T (in significant linkage disequilibrium [LD] with _UGT1A1*28_ ), with reported positive predictive value ranging from 20% to 60% depending on race/ethnicity. Thus, race/ethnicity may modify the genetic effect (see Figure 1 in the original guideline document). Nongenetic factors such as fasting and diet can also affect bilirubin concentrations.

This CPIC guideline assumes that the _UGT1A1_ genotype results are already available to the prescriber. It is beyond the scope of this guideline to make recommendations regarding whether or not _UGT1A1_ genotyping should be performed.

# Implementation of the Guideline

## Description of Implementation Strategy



The Supplementary Material (see the "Availability of Companion Documents" field) contains example clinical decision support (CDS) tools that can be used within electronic health records (EHRs) to assist clinicians in applying genetic information to patient care for the purpose of drug therapy optimization. Clinical implementation resources include cross-references for drug and gene names to widely used terminologies and standardized nomenclature systems (see Supplementary Tables S5 and S6), workflow diagram (see Supplemental Figures S1 and S2), and example text for documentation in the EHR and point-of care alerts (see Supplementary Tables S7 and S8).

## Implementation Tools

- Clinical Algorithm
- Resources

# Institute of Medicine (IOM) National Healthcare Quality Report Categories

## IOM Care Need

- Living with Illness

## IOM Domain

- Effectiveness
- Safety

# Identifying Information and Availability

## Bibliographic Source(s)

- Gammal RS, Court MH, Haidar CE, Iwuchukwu OF, Gaur AH, Alvarellos M, Guillemette C, Lennox JL, Whirl-Carrillo M, Brummel SS, Ratain MJ, Klein TE, Schackman BR, Caudle KE, Haas DW. Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for UGT1A1 and atazanavir prescribing. Clin Pharmacol Ther. 2016 Apr;99(4):363-9. [40 references] [ PubMed ](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=26417955)

## Adaptation



Not applicable: The guideline was not adapted from another source.

## Date Released

2016 Apr

## Guideline Developer(s)

- Clinical Pharmacogenetics Implementation Consortium - Independent Expert Panel

## Source(s) of Funding



This work was funded by the National Institutes of Health (NIH) PAAR4Kids (UO1 GM92666), PharmGKB (R24 GM61374), U01 HL0105198, P30 AI110527, R01 AI077505, TR000445 (to D.W.H.), and R01 GM102130 (to M.H.C.).

## Guideline Committee



Not stated

## Composition of Group That Authored the Guideline



 _Authors_ : RS Gammal, Department of Pharmaceutical Sciences, St. Jude Children's Research Hospital, Memphis, Tennessee, USA; MH Court, Individualized Medicine Program, Department of Veterinary Clinical Sciences, Washington State University College of Veterinary Medicine, Pullman, Washington, USA; CE Haidar, Department of Pharmaceutical Sciences, St. Jude Children's Research Hospital, Memphis, Tennessee, USA; OF Iwuchukwu, Division of Pharmaceutical Sciences, Fairleigh Dickinson University School of Pharmacy, Florham Park, New Jersey, USA, Division of Clinical Pharmacology, Department of Medicine, Vanderbilt University Medical Center, Nashville, Tennessee, USA; AH Gaur, Department of Infectious Diseases, St. Jude Children's Research Hospital, Memphis, Tennessee, USA; M Alvarellos, Department of Genetics, Stanford University, Stanford, California, USA; C Guillemette, Laval University CHU de Quebec Research Center, Quebec, Quebec, Canada; JL Lennox, Division of Infectious Disease, Emory University School of Medicine, Atlanta, Georgia, USA; M Whirl-Carrillo, Department of Genetics, Stanford University, Stanford, California, USA; SS Brummel, Center for Biostatistics in AIDS Research, Harvard T.H. Chan School of Public Health, Boston, Massachusetts, USA; MJ Ratain, Center for Personalized Therapeutics, Comprehensive Cancer Center, The University of Chicago, Chicago, Illinois, USA; TE Klein, Department of Genetics, Stanford University, Stanford, California, USA; BR Schackman, Department of Healthcare Policy and Research, Weill Cornell Medical College, New York, New York, USA; KE Caudle Department of Pharmaceutical Sciences, St. Jude Children's Research Hospital, Memphis, Tennessee, USA; DW Haas, Departments of Medicine, Pharmacology, Pathology, Microbiology & Immunology, Vanderbilt University School of Medicine, Nashville, Tennessee, USA.

## Financial Disclosures/Conflicts of Interest



D.W.H. has been a consultant to Merck. All other authors declare no conflicts.

## Guideline Status



This is the current release of the guideline.

This guideline meets NGC's 2013 (revised) inclusion criteria.

## Guideline Availability



Available from the [Clinical Pharmacogenetics Implementation Consortium (CPIC) Web site](https://cpicpgx.org/guidelines/guideline-for-atazanavir-and-ugt1a1/ "Clinical Pharmacogenetics Implementation Consortium \(CPIC\) Web site").

## Availability of Companion Documents



The following are available:

  * Supplementary material, including tables, methodological information, and implementation resources, is available from the [Clinical Pharmacogenetics Implementation Consortium (CPIC) Web site](https://cpicpgx.org/guidelines/guideline-for-atazanavir-and-ugt1a1/ "Clinical Pharmacogenetics Implementation Consortium \(CPIC\) Web site"). 
  * A UGT1A1 frequency table is available from the [CPIC Web site](https://cpicpgx.org/guidelines/guideline-for-atazanavir-and-ugt1a1/ "CPIC Web site"). 



## Patient Resources



None available

## NGC Status



This NGC summary was completed by ECRI Institute on September 23, 2016. The information was verified by the guideline developer on November 2, 2016.

## Copyright Statement



This NGC summary is based on the original guideline, which is subject to the guideline developer's copyright restrictions.

CPIC is a shared project between PharmGKB and the Pharmacogenomics Research Network. This guideline is published in Clinical Pharmacology and Therapeutics (CPT), an American Society for Clinical Pharmacology and Therapeutics journal (ASCPT). Electronic access to the CPIC guidelines is available at PharmGKB, [cpicpgx.org](http://cpicpgx.org "CPIC Web site"), and through PubMed Central.

CPICÂ® is a registered service mark of the [U.S. Department of Health & Human Services (HHS)](http://www.hhs.gov/ "DHHS Web site").

# Disclaimer

## NGC Disclaimer



The National Guideline Clearinghouseâ„¢ (NGC) does not develop, produce, approve, or endorse the guidelines represented on this site.

All guidelines summarized by NGC and hosted on our site are produced under the auspices of medical specialty societies, relevant professional associations, public or private organizations, other government agencies, health care organizations or plans, and similar entities.

Guidelines represented on the NGC Web site are submitted by guideline developers, and are screened solely to determine that they meet the [NGC Inclusion Criteria](/help-and-about/summaries/inclusion-criteria).

NGC, AHRQ, and its contractor ECRI Institute make no warranties concerning the content or clinical efficacy or effectiveness of the clinical practice guidelines and related materials represented on this site. Moreover, the views and opinions of developers or authors of guidelines represented on this site do not necessarily state or reflect those of NGC, AHRQ, or its contractor ECRI Institute, and inclusion or hosting of guidelines in NGC may not be used for advertising or commercial endorsement purposes.

Readers with questions regarding guideline content are directed to contact the guideline developer.

